Microsoft.Toolkit.Uwp.SampleApp/readme.md

```diff
@@ -35,9 +35,10 @@ Here is an example:
 ```
 
 You can define "interactive" values in this file. The value types can be:
-* String: You want the user to provide a text. The string is built like this @[Name:**String**:Default value]
+* String: You want the user to provide a text. The string is built like this @[Name:**String**:Default value:UpdateSourceTriggerId]
 * Slider: You want the user to provide a double value. The string is built like this @[Name:**Slider**:Default value:min-max]
 * DoubleSlider: Same as slider but with double values (0.01 precision)
+* RangeSelectorMin and RangeSelectorMax: You want the user to provide two values, usualy for max and min. The strings are built like this @[NameMin:RangeSelectorMin:Default min value:min:ID]@ and @[NameMax:RangeSelectorMax:Default max value:max:ID]@. The IDs need to match.
```

Comment:
You don't need the extra `@` at the end in this example, as that's for two-way binding explained later.

Reply:
Done.